activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerITSupport.java

```diff
@@ -26,14 +29,15 @@
     private ObjectMapper objectMapper;
 
     /**
-     * This is not a test. It's actually generating the swagger.json and yaml definition of the service.
-     * It is used by maven generate-swagger profile build.
+     * This is not a test. It's actually generating the swagger.json and yaml definition of the
+     * service. It is used by maven generate-swagger profile build.
      */
     @Test
     public void generateSwagger() throws Exception {
         mockMvc.perform(get("/v2/api-docs").accept(MediaType.APPLICATION_JSON))
             .andDo((result) -> {
-                JsonNode jsonNodeTree = objectMapper.readTree(result.getResponse().getContentAsByteArray());
+                JsonNode jsonNodeTree = objectMapper
+                    .readTree(result.getResponse().getContentAsByteArray());
```

Comment:
What's the codestyle you are using? I assume that you are having a hard wrap at 100, but that's just doesn't play well with 4 space indents as things start to hard wrap very quickly and annoyingly unnecessarily, as even the github diff viewer can work with 120 or even more.

I will have a PR in the activiti project fixing those misunderstanding as soon as I have some spare time

Reply:
Good question. I'm sure I had an "Activiti/Alfresco" formatting conf inside IDEA, but now it isn't in place anymore. Maybe I've lost it updating the IDE.

Reply:
Just reimported the checkstyle configuration ffrom Activiti, let's reformat

Reply:
So the "Activiti/Alfresco" formatting is deprecated, we are using the Google one as a base, just that we have 4 space indents instead of 2 space indents. All you need to do in my view is to set the hard-wrap from the default 100 to anything between 120 and 140